rfcs/2022-04-09-accessibility.md

```diff
@@ -0,0 +1,167 @@
+---
+Start Date: 08-04-2022
+RFC PR: (leave this empty if no PR yet)
+Blade Issue: (leave this empty if no issue yet)
+---
+
+# Blade Accessibility <!-- omit in toc -->
+
+### Table Of Contents <!-- omit in toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+- [Detailed Design](#detailed-design)
+- [Drawbacks/Constraints](#drawbacksconstraints)
+- [Alternatives](#alternatives)
+- [Adoption strategy](#adoption-strategy)
+- [How do we educate people?](#how-do-we-educate-people)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+# Summary
+
+
+## What is Accessibility?
+
+Accessibility is the practice of making websites usable by people of all abilities and disabilities.   
+This includes people with visual, auditory, physical, and cognitive impairments, as well as those with temporary disabilities such as broken bones, color blindness, and temporary loss of vision, but the practice of making sites accessible also benefits other groups such as people using mobile phones, people with temporary disabilities, situational limitations.
+
```

Comment:
Commonly I see people confuse a11y with something that isn't important for everyone. Perhaps we should add a note on how / why it's crucial for everything - for example filling forms, closing modal, etc.

Reply:
Correct, will add few lines to highlight why accessibility is for everyone.

Reply:
Will add this video https://www.youtube.com/watch?v=3f31oufqFSM